### ISSUE TITLE
khard doesn't support quoted-printable encoding

### ISSUE THREAD

**Issue by vbwagner** [scheibler/khard#183]
Some phones encode non-ascii characters (including german, but I'm more concerned with cyrillic) as quoted-printable, adding ENCODING=QUOTED-PRINTABLE into name of the field.
khard version 0.11.3 from Debian stretch chockes on such VCF files and tells that it cannot parse encoding field

Here is an example of such a vcf file, exported from some Sony XPeria phone. (I've just replaced real phone nubmer with all zeros)

```
BEGIN:VCARD
VERSION:2.1
N;CHARSET=UTF-8;ENCODING=QUOTED-PRINTABLE:;=D0=9A=D0=BE=D0=BC=D0=BC=D0=B5=D1=80=D1=87=D0=B5=D1=81=D0=BA=D0=B8=D0=
=B9=20=D0=90=D0=B2=D1=82=D0=BE=D0=B1=D1=83=D1=81=20=D0=9C=D0=BE=D1=81=
=D0=BA=D0=B2=D0=B0=2D=D0=9E=D1=81=D1=82=D0=B0=D1=88=D0=BA=D0=BE=D0=B2=
;;;
FN;CHARSET=UTF-8;ENCODING=QUOTED-PRINTABLE:=D0=9A=D0=BE=D0=BC=D0=BC=D0=B5=D1=80=D1=87=D0=B5=D1=81=D0=BA=D0=B8=D0=
=B9=20=D0=90=D0=B2=D1=82=D0=BE=D0=B1=D1=83=D1=81=20=D0=9C=D0=BE=D1=81=
=D0=BA=D0=B2=D0=B0=2D=D0=9E=D1=81=D1=82=D0=B0=D1=88=D0=BA=D0=BE=D0=B2=

TEL;CELL:+70000000000
END:VCARD
```

**Comment by lucc** [comment]
I can veryfy your bug but sadly that is a problem with the https://github.com/eventable/vobject library. And if I remember correctly they only support vcard version >=3.

Maybe you can open a bug report with them?

**Comment by lucc** [comment]
Acording to the [RFC](https://tools.ietf.org/html/rfc2426#section-5) quoted-printable is only available in vCard version 2.1 and not from 3 onwards.  We use the [vobject library](https://github.com/eventable/vobject) which itself does not support version 2.1.  You can open a pull request with them to add support for it, [as mentioned here](https://github.com/eventable/vobject/issues/118).

Therefore I will close this as wontfix (here).
